metricbeat/module/postgresql/replication/_meta/fields.yml

```diff
@@ -0,0 +1,83 @@
+- name: replication
+  type: group
+  description: >
+    One row per replication, showing database replication statistics. Collected by querying
+    pg_stat_database.
+  release: ga
+  fields:
+    - name: write_lag
+      type: long
+      description: >
+        Elapsed time during committed WALs from primary to the replica.
+    - name: state
+      type: keyword
+      description: >
+        WAL sender state.
+    - name: sync_priority
+      type: long
+      description: >
+        Priority of replica server being chosen as a synchronous replica.
+    - name: backend_start
+      type: long
+      description: >
+        Start time when replica connected to the primary.
+    - name: replay_lag
+      type: long
+      description: >
+        Elapsed time during committed WALs from primary to the replica.
+        Fully committed to replica node.
+    - name: flush_lag
+      type: long
+      description: >
+        The elapsed time during committed WALs from primary to the replica.
+        WALs have already been flushed but have not yet been applied.
+    - name: pid
+      type: long
+      description: >
+        Process id of walsender process.
+    - name: client.address
+      type: long
+      description: >
+        Address of replica/streaming replication.
+    - name: client.port
+      type: long
+      description: >
+        Port of replica/streaming replication.
+    - name: client.hostname
+      type: long
+      description: >
+        Hostname of replica/streaming replication.
+    - name: replay_lsn
+      type: long
+      description: >
+        Last transaction that has been replayed at the replica.
+    - name: user.name
+      type: long
+      description: >
+        Name of the user which is used for streaming replication.
+    - name: user.id
+      type: long
+      description: >
+        ID of the user which is used for streaming replication.
+    - name: sync_state
+      type: long
+      description: >
+        Sync state of the replica.
+    - name: flush_lsn
+      type: long
+      description: >
+        Last transaction flush on disk at the replica.
+    - name: application_name
+      type: long
+      description: >
+        Number of queries cancelled due to conflicts with recovery in this
+        database.
```

Comment:
`application_name` is of type long? Also, the description is not correct. Please vet all names, type and description once again.

Isn't application_name is the name of the application that is connected to this WAL sender? It's also of type string.